### PR TITLE
HEADER

### PR DESCRIPTION
**Pull request by VladislavApelhants**

### CODE REVIEW

**Review by yuriiSY — APPROVED** [review]

**Comment by VladislavApelhants** [comment]
efwefwe

**Review by kamineckav — COMMENTED** [review]
Okay!

**Review by kamineckav — APPROVED** [review]
Okay!

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.header{
+    padding: 30px 0;
```
Comment (vladvolchek) [inline css/styles.css]: padding 40px

**Review by vladvolchek — CHANGES_REQUESTED** [review]

File: css/styles.css (hunk 1 above)
Comment (VladislavApelhants) [reply css/styles.css]: ok

**Review by vladvolchek — APPROVED** [review]

**Pull request merged**